src/Chainweb/Version.hs

```diff
@@ -755,12 +736,7 @@ oldTargetGuard _ _ = False
 -- live chains, enforcing the following condition must be ignored for the
 -- historical blocks for which both the Nonce and Flags could be anything.
 --
-skipFeatureFlagValidationGuard
-    :: ChainwebVersion
-    -> BlockHeight
-        -- ^ height of header
-    -> Bool
+skipFeatureFlagValidationGuard :: ChainwebVersion -> BlockHeight -> Bool
 skipFeatureFlagValidationGuard Mainnet01 h = h < 530500  -- ~ 2020-05-01T00:00:xxZ
-skipFeatureFlagValidationGuard Development _ = True
-skipFeatureFlagValidationGuard Testnet04 _ = False
+skipFeatureFlagValidationGuard Development h = h < 140
```

Comment:
This causes validation failures. It's here only for testing the respective mainnet behavior. Those tests have been completed by now.

```suggestion
```